src/screens/bottom-sheet/dashboard-menu-bottom-sheet/dashboard-menu-bottom-sheet.tsx

```diff
@@ -0,0 +1,102 @@
+    public componentDidMount() {
+        this.bottomSheet.current.snapTo(BOTTOM_SHEET_HEIGHT);
+    }
+
+    public renderBottomSheetContent = () => {
+        const { styles } = this.props;
+
+        return (
+            <View style={[styles.content, { height: BOTTOM_SHEET_HEIGHT }]}>
+                <TouchableOpacity style={styles.rowContainer}>
+                    <View style={styles.iconContainer}>
+                        <Icon name="archive-locker" size={ICON_SIZE} style={styles.icon} />
+                    </View>
+                    <View style={styles.textContainer}>
+                        <Text style={styles.title}>
+                            {translate('DashboardMenu.transactionHistory')}
+                        </Text>
+                        <Text style={styles.description}>
+                            {translate('DashboardMenu.description')}
+                        </Text>
+                    </View>
+                </TouchableOpacity>
+
+                <TouchableOpacity style={styles.rowContainer}>
+                    <View style={styles.iconContainer}>
+                        <Icon name="pencil" size={ICON_SIZE} style={styles.icon} />
+                    </View>
+                    <View style={styles.textContainer}>
+                        <Text style={styles.title}>{translate('DashboardMenu.manageAccount')}</Text>
+                        <Text style={styles.description}>
+                            {translate('DashboardMenu.description')}
+                        </Text>
+                    </View>
+                </TouchableOpacity>
+
+                <TouchableOpacity style={styles.rowContainer}>
+                    <View style={styles.iconContainer}>
+                        <Icon name="flash" size={ICON_SIZE} style={styles.icon} />
+                    </View>
+                    <View style={styles.textContainer}>
+                        <Text style={styles.title}>{translate('DashboardMenu.walletConnect')}</Text>
+                        <Text style={styles.description}>
+                            {translate('DashboardMenu.description')}
+                        </Text>
+                    </View>
+                </TouchableOpacity>
+            </View>
+        );
+    };
+
+    public renderHeader = () => (
+        <View style={this.props.styles.header}>
+            <Icon name="arrow-button-down" size={28} style={this.props.styles.icon} />
+        </View>
+    );
+
+    public render() {
+        return (
+            <View style={this.props.styles.container}>
+                <BottomSheet
+                    ref={this.bottomSheet}
+                    initialSnap={0}
+                    snapPoints={[0, BOTTOM_SHEET_HEIGHT]}
+                    renderContent={this.renderBottomSheetContent}
+                    renderHeader={this.renderHeader}
+                    onCloseStart={() => setTimeout(() => this.props.onClose(), 500)} // TODO: fix here, onCloseEnd it's not working properly
```

Comment:
I okish with this, but we need to test it very well. 
I found this issue (https://github.com/osdnk/react-native-reanimated-bottom-sheet/issues/136) is it relevant for our case ?

Reply:
Yes indeed, this issue fixed our problems. Awesome  🎉